sqldelight-core/src/main/kotlin/com/squareup/sqldelight/core/lang/psi/StmtIdentifierMixin.kt

```diff
@@ -16,7 +24,19 @@ abstract class StmtIdentifierMixin(
     SqliteAnnotatedElement {
   override fun getName() = identifier()?.text
 
-  override fun setName(p0: String): PsiElement {
+  override fun setName(name: String): PsiElement {
+    val parserDefinition = LanguageParserDefinitions.INSTANCE.forLanguage(language) as SqliteParserDefinition
+    var builder = PsiBuilderFactory.getInstance().createBuilder(
+        project, node, parserDefinition.createLexer(project), language, name
+    )
+    builder = GeneratedParserUtilBase.adapt_builder_(SqliteTypes.IDENTIFIER, builder, SqliteParser(),
+        SqliteParser.EXTENDS_SETS_
+    )
```

Comment:
Weird wrapping

Reply:
This is done by the Square Formatter and should be if possible changed there